src/bin/server.ts

```diff
@@ -10,10 +10,16 @@ interface Args {
   _: Array<string>;
   config?: string;
   "mock-driver": boolean;
+  "websocket-port": number;
```

Comment:
```suggestion
  "websocket-port"?: number;
```
I think... Would need someone else to confirm

Reply:
Just tested it, it's not needed

Reply:
For what it is worth it worked without making that optional but I'm admittedly a novice when it comes to typescript so if it is still considered poor practice I can update it.